home/home2.2/imageChanger.java

```diff
@@ -0,0 +1,154 @@
+import java.util.ArrayList;
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.BufferedWriter;
+class Constants {
+    public static final int MatrixSize = 16;
+}
+class Pixel
+{
+    private String color;
```

Comment:
I hope the same improvements from c++ can be applied here like integer fields r g b instead of string and so on